lib/graphql/schema.rb

```diff
@@ -47,6 +48,7 @@ module GraphQL
   #
   class Schema
     include BuildFromDefinition
+    include SchemaComparator
```

Comment:
I see a couple of includes have found their way here already, but how about implementing methods to call through to the underlying class instead of including a module? For example: 

```ruby 
def self.compare(s1, s2) 
  SchemaComparator.find_changes(s1, s2) 
end 

def compare_to(s2) 
  self.class.compare(self, s2)
end 
```

The advantages I see are:

- The reader knows exactly what methods on `Schema` come from `SchemaComparator` 
- Someone who modifies `SchemaComparator` can easily find its usages in `Schema` 

Would that work in this case?